src/config_resolver.go

```diff
@@ -16,10 +16,11 @@ package main
 import (
 	"bytes"
 	"encoding/json"
-	"github.com/elodina/go-avro"
 	"os"
 	"text/template"
 	"time"
+
+	"github.com/elodina/go-avro"
```

Comment:
To keep everything consistent can you run: `make format`

This ensures that everything is in keeping with the expected Go formatting standards.